ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -204,63 +174,55 @@ export const AdminUserTable = withUserProfile()(
-          user,
-          'complianceTraining'
-        ),
-        ctComplianceTraining: this.accessModuleCellContents(
-          user,
-          'ctComplianceTraining'
-        ),
-        contactEmail: (
-          <a href={`mailto:${user.contactEmail}`}>{user.contactEmail}</a>
-        ),
-        dataAccess: this.dataAccessContents(user),
-        dataUseAgreement: this.accessModuleCellContents(
-          user,
-          'dataUseAgreement'
-        ),
-        enabled: user.disabled ? <DisabledIcon /> : <EnabledIcon />,
-        eraCommons: this.accessModuleCellContents(user, 'eraCommons'),
-        rasLinkLoginGov: this.accessModuleCellContents(user, 'rasLinkLoginGov'),
-        firstSignInTime: this.formattedTimestampOrEmptyString(
-          user.firstSignInTime
-        ),
-        firstSignInTimestamp: user.firstSignInTime,
-        institutionName: this.displayInstitutionName(user),
-        name: (
-          <AdminUserLink username={user.username} target='_blank'>
-            {user.familyName + ', ' + user.givenName}
-          </AdminUserLink>
-        ),
-        // used for filter and sorting
-        nameText: user.familyName + ' ' + user.givenName,
-        twoFactorAuth: this.accessModuleCellContents(user, 'twoFactorAuth'),
-        username: (
-          <AdminUserLink username={user.username} target='_blank'>
-            {usernameWithoutDomain(user.username)}
-          </AdminUserLink>
-        ),
-      }));
+    convertProfilesToFields(users: AdminTableUser[]): DataTableFields[] {
+      return users
+        .map((user) => ({
+          // 'text' fields are used for sorting
+
+          name: (
+            <AdminUserLink username={user.username} target='_blank'>
+              {user.familyName + ', ' + user.givenName}
+            </AdminUserLink>
+          ),
+          nameText: user.familyName + ' ' + user.givenName,
+
+          username: (
+            <AdminUserLink username={user.username} target='_blank'>
+              {usernameWithoutDomain(user.username)}
+            </AdminUserLink>
+          ),
+          usernameText: usernameWithoutDomain(user.username),
+
+          contactEmail: (
+            <a href={`mailto:${user.contactEmail}`}>{user.contactEmail}</a>
+          ),
+          contactEmailText: user.contactEmail,
+
+          institutionName: this.displayInstitutionName(user),
+          institutionNameText: user.institutionName,
+
+          enabled: user.disabled ? <DisabledIcon /> : <EnabledIcon />,
+          dataAccess: this.dataAccessContents(user),
+          bypass: (
+            <AdminUserBypass
+              user={{ ...user }}
+              onBypassModuleUpdate={() => this.loadProfiles()}
+            />
+          ),
+
+          firstSignInTime: this.formattedTimestampOrEmptyString(
+            user.firstSignInTime
+          ),
+        }))
+        .sort((f1: DataTableFields, f2: DataTableFields) =>
+          f1.nameText.localeCompare(f2.nameText)
```

Comment:
passing in `sortField` to `DataTable` was not working correctly.  This sets an initial sort by username, and the sort field can be changed by clicking the table